.github/workflows/legacy-tests.yml

```diff
@@ -50,6 +50,11 @@ jobs:
           cache: npm
           check-latest: true
         if: '${{!steps.release-check.outputs.IS_RELEASE}}'
+      - name: Install PNPM
+        run: |
+          corepack enable
```

Comment:
corepack is only available as of 14.19, if i read the docs right, but we test with 14.18. Why does this work?

Reply:
It got backported ;)

Reply:
But only pnpm version 7 works with node 14